phpstan.neon

```diff
@@ -3,14 +3,14 @@ includes:
 
 parameters:
   level: 0
-  autoload_files:
-    - %rootDir%/../../../autoload.php
-#  autoload_directories:
-#    - %rootDir%/../../../vendor/bin/.phpunit/phpunit-5.7/src
+  paths:
+    - src
+  bootstrapFiles:
```

Comment:
Removing `autoload_files` since they switched to custom symbols and `bootstrapFiles`: https://phpstan.org/user-guide/discovering-symbols#custom-autoloader